platform/golden/image-cd.pkr.hcl

```diff
@@ -35,13 +35,21 @@ variable "disk_size" {
   default = "200000"
 }
 
+###
+#
+# Please note target_path is relative to ${PWD}
+
+variable "target_path" {
+  type    = string
+  default = "build/"
```

Comment:
I think this should be "build" in order to avoid double slashes during interpolation. However, it's not currently a problem, just a difference.

Reply:
Historically have done this for security reasons. When the target path is joined with other arguments, no leading slash is required in the variable contents.